### ISSUE TITLE
[Bug]:  MudTextField not clickable

### ISSUE THREAD

**Issue by Detrua** [fgilde/MudBlazor.Extensions#69]
### Contact Details

eric.lehner@dataunit.ch

### What happened?

When using:

```c#
var options = new DialogOptionsEx
        {
            NoHeader = true,
            DragMode = MudDialogDragMode.Simple
        };
```

I can click everywhere in Dialog and it is draggable.
I cannot click into Textfields anmore.
Dropdown are clickable.

### Expected Behavior

Text fields are clickable and I can fill them with text.

### What application type are you referring to?

ServerRendered

### MudBlazor.Extension Version

1.7.83

### MudBlazor Version

6.12.0

### What browser are you using?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fgilde** [comment]
Thanks for reporting. Was able to reproduce and will try to fix as soon as possible

**Comment by fgilde** [comment]
Sorry sorry for long delay, i was very busy the last days. However I fixed your described bug. Will be released with version 1.7.84 but you can test it with the preview package [1.7.84-prev-240209113-main](https://www.nuget.org/packages/MudBlazor.Extensions/1.7.84-prev-240209113-main)
